include/z64transition.h

```diff
@@ -4,24 +4,31 @@
 #include "ultra64.h"
 #include "color.h"
 
+typedef enum {
+    /* 0 */ TRANS_TILE_OFF,
+    /* 1 */ TRANS_TILE_SETUP,
+    /* 2 */ TRANS_TILE_INIT,
+    /* 3 */ TRANS_TILE_PROCESS
+} TransitionTileStatus;
```

Comment:
this heavily mirrors the names given to the pause background state given in https://github.com/zeldaret/oot/pull/1368/files
which notably is still under debate. so ideally we come to an agreement there before we start introducing things like it. we need to just get that conversation started again and pull it back out of limbo tho

Reply:
please do <img src="https://cdn.discordapp.com/emojis/946304013832437810.webp?size=128&quality=lossless" alt="despair" height="24">

Reply:
no, I thought they were the same until you pointed that out just now lol